### DISCUSSION TITLE
Is there any way to link to origin git repository?

### DISCUSSION THREAD

**Discussion by Destroy666x** [JanDeDobbeleer/oh-my-posh#4215] [Troubleshoot] [answered]
There's `UpstreamUrl` for `upstream`, is there any way to link `origin` instead?

**Comment by JanDeDobbeleer** [comment]
@Destroy666x `upstream` is `origin` if that's the remote name.

**Reply by Destroy666x** [reply]
I mean I got both `upstream` and `origin` - classic main repo + fork. Is there any way to link the `origin` then (or maybe even both)?

**Reply by JanDeDobbeleer** [reply]
@Destroy666x upstream changes depending on your branch' upstream reference. Is there a use-case to have access to all remotes?

**Reply by Destroy666x** [reply]
Personally I'm (and I've been) working on projects that use forks as side path of a project and you interact with origin and upstream a lot. So e.g. linking both for quick opening repo in the browser and check for example PRs would be the use case I'm looking for.

**Reply by JanDeDobbeleer** [reply, marked as answer]
@Destroy666x forgot to mention this but there's a hidden `Remotes` map now of type `map[string]string` which implies that you can use `Remotes.upstream` for example. It uses the name of the remote.

**Reply by Destroy666x** [reply]
Thanks, I'll try

EDIT: works fine
